### PR TITLE
Note on bindToScope in Apple/Cocoa custom instr.

### PR DESCRIPTION
**Pull request by realkosty**
## Pre-merge checklist

*If you work at Sentry, you're able to merge your own PR without review, but please don't unless there's a good reason.*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

## Description of changes

*Describe your changes here. If your PR relates to or resolves an issue, add a link to that too.*

## Legal Boilerplate

Look, I get it. The entity doing business as "Sentry" was incorporated in the State of Delaware in 2015 as Functional Software, Inc. and is gonna need some rights from me in order to utilize my contributions in this here PR. So here's the deal: I retain all rights, title and interest in and to my contributions, and by keeping this boilerplate intact I confirm that Sentry can use, modify, copy, and redistribute my contributions, under Sentry's choice of terms.

## Extra resources

- [Sentry Docs contributor guide](https://docs.sentry.io/contributing/)

### CODE REVIEW

File: src/platform-includes/performance/create-transaction-bound-to-scope/apple.mdx [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 Our SDK can bind a transaction to the scope making it accessible to every method running within this scope by calling `SentrySDK#startTransaction` method with `bindToScope` parameter to `true`.
 
+<Note>
```
Comment (shanamatthews) [inline src/platform-includes/performance/create-transaction-bound-to-scope/apple.mdx:5]:
```suggestion
```

File: src/platform-includes/performance/create-transaction-bound-to-scope/apple.mdx [hunk 2]
```diff
@@ -2,6 +2,12 @@
...
+
+`bindToScope` additionally ensures that your new transaction replaces any one that may be already started. This is useful if you want custom instrumentation to co-exist with auto-instrumented transactions.
+
+</Note>
```
Comment (shanamatthews) [inline src/platform-includes/performance/create-transaction-bound-to-scope/apple.mdx:9]:
```suggestion
```

**Review by shanamatthews — APPROVED** [review]
Also looks good to me!

**Comment by shanamatthews** [comment]
@philipphofmann - could you double check this PR?

**Review by philipphofmann — APPROVED** [review]
LGTM.

**Pull request merged**